.github/workflows/ci-demo-vs.yml

```diff
@@ -0,0 +1,50 @@
+name: Continuous Integration for Demo vs
+# This workflow is triggered on pull requests to the main branch and on pushes to the main branch.
+# It runs a series of steps to validate the code, including checking out the code, setting up Node.js, installing dependencies, building the project, and checking formatting and types.
+
+on:
```

Comment:
Do we really need to add a specific CI workflow for this particular demo service?